client/securedrop_client/export_status.py

```diff
@@ -0,0 +1,64 @@
+from enum import Enum
+
+
+class ExportError(Exception):
+    def __init__(self, status: "ExportStatus"):
+        self.status: "ExportStatus" = status
+
+
+class ExportStatus(Enum):
+    """
+    All possible strings returned by the qrexec calls to sd-devices. These values come from
+    `print/status.py` and `disk/status.py` in `securedrop-export`
+    and must only be changed in coordination with changes released in that component.
+    """
```

Comment:
Suggestion for future refactoring:  This suggests to me that we need a package `securedrop-export-common`, on which both `securedrop-client` and `securedrop-export` depend.

Reply:
Filed as #1855.